packages/testing/src/TestServer.ts

```diff
@@ -10,6 +10,9 @@ export class ColyseusTestServer {
     create: Client['create'],
     joinById: Client['joinById'],
     reconnect: Client['reconnect'],
+    registerWithEmailAndPassword: Client['auth']["registerWithEmailAndPassword"];
+    signOut: Client['auth']["signOut"];
+    signInWithEmailAndPassword: Client['auth']["signInWithEmailAndPassword"];
```

Comment:
Can we have the entire `Client['auth']` reference here under `auth` instead of binding the individual `auth.*` methods?

Reply:
@endel Done ;)